content/50.discussion.md

```diff
@@ -11,11 +11,8 @@ In this review, we seek to explain the scientific rationale underlying these tec
 
 ### Concerns about Equity in Healthcare
 
-Scientific and medical research broadly is shaped by a number of biases, such as Eurocentrism, that are of significant concern in the face of a global pandemic.
-Some concerns include: 
-1. Who is being included in clinical trials?
-2. While biotechnological developments are exciting, how likely are they to be available to people broadly?
-3. Even this review is likely biased (e.g., we look at traditional western medicine in the nutraceuticals section and mainly report statistics from US and other data collection services)
+Scientific and medical research broadly is shaped by a number of biases.
+Some concerns include how clinical trials recruit and operate.
```

Comment:
This is specifically what @tlukan is researching right now so leaving this to mark that it's in progress.

Reply:
Hello!  The private message had been to say that I was partly in agreement with @yemarshall 's concerns that the terms "Eurocentrism" and "traditional Western medicine" were very vague here and could be badly misinterpreted.  It's true that most of the research was done in Europe, the USA, or China, but including the USA in Europe would be stretching the term, and including China as part of Europe would stretch the term so far as to snap it entirely.
And more importantly, people use references to "non-western medicine" to mean many different things, some of them helpful (like ethnobotany) and some unhelpful (like crystal healing or something on YouTube).  
On the other hand, it was just an outline, so it's supposed to be vague.
So maybe it would be helpful here to just make a list of exactly what things we do want to say.  @tlukan would of course have to add to it.
To me, the most important things I'm worried about regarding international issues are:
1. The pandemic is starting to take off in Africa and I'm pretty scared about it because some places don't have the resources to deal with it.
2. The pandemic has caused enormous problems in Brazil and other South and Central American countries, but there hasn't been much mention of it in the scientific literature.
3. In some ways, the research done in USA and China will apply equally well to other places (the molecular structure of the virus is the same all over) but in other ways, every region will need to make its own decisions about how to roll out culturally appropriate public health solutions. 
4. We hope to have more solutions to review in the next revision.
Regarding domestic issues in the USA, we already mentioned that African-Americans, Native Americans and Hispanics are very hard-hit by the virus, so the main thing I would want to mention in the discussion are that we need to know more about why, and we need more effort to mitigate the harms in these populations.
Thank you so much!

Reply:
Also, I'm copying and pasting @yemarshall 's suggestions from issue #514:

"Considerations for equity in health
One of the strategies and goals of effective science and medical research is to avoid and remove biases. Also, identifying specific areas or aspects where health can be improved is a public health strategy which can both enhance program effectiveness directly in the area or aspect identified and indirectly in the whole population because infectious disease spreads and epidemics/pandemics are experienced by an interacting population. In the face of a global pandemic, some important considerations include the following: 1. What aspects of data are being collected? 2. Who is being included in clinical trials? 3. How do we make sure biotechnologies are widely available? 4. How do we improve consideration of health inequities in research and reviews, such as this one?"